packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -33,26 +32,34 @@ const DEFAULT_OPTIONS = {
   userAvatarInitials: '',
 
   // Bubble
-  bubbleBackground: 'White',
```

Comment:
Removing style options is a breaking change. We want to support the legacy options as well as the new ones. Instead of removing, `bubble*`, the values should be set to `undefined`, and then in the style options we should use `bubble*` if it's not falsy; otherwise, use `bubbleFromBot*`.